configure/cpack_win.cmake

```diff
@@ -1,33 +1,20 @@
+set(COMPILER_TAG ${CMAKE_VS_PLATFORM_TOOLSET})
```

Comment:
For some reason `CMAKE_VS_PLATFORM_TOOLSET` was not used with `MSVC90`, but now it is. Are you sure it is defined? The documentation only mentions that: *VS 10 and above use MSBuild under the hood ...*

https://cmake.org/cmake/help/latest/variable/CMAKE_VS_PLATFORM_TOOLSET.html

Reply:
Yes, it is undefined. Just see the logs:
```
CPack: - package: C:/projects/static_build/libtango_9.3.4__x64_static_release.msi generated
```

Other jobs print version here:
```
CPack: - package: C:/projects/static_build/libtango_9.3.4_v141_x64_static_debug.msi generated.
```

Reply:
Thanks for carerful checking. I'll find a workaround.

Reply:
@mliszcz Done.